lib/utils/BvEvent.js

```diff
@@ -23,7 +23,9 @@ export class BvEvent {
         let defaultPrevented = false;
         // Recreate preventDefault method. One way setter.
         this.preventDefault = function preventDefault() {
-            defaultPrevented = true;
+            if (this.cancelable) {
```

Comment:
Although these are properties of the event object, not the component, so true is not as bad.